gammapy/irf/background.py

```diff
@@ -114,3 +115,17 @@ def read(cls, filename, hdu='BACKGROUND'):
         filename = make_path(filename)
         hdulist = fits.open(str(filename))
         return cls.from_hdulist(hdulist, hdu=hdu)
+
+    def to_table(self):
+        #TODO: BGD or BKG - see https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/bkg/index.html
+        table = Table()
+        table['DETX_LO'] = [self.data.axis('detx').hi]*self.data.axis('detx').unit
```

Comment:
copy & paste error on the line filling `table['DETX_LO']`: should be `self.data.axis('detx').lo`, you have `self.data.axis('detx').hi`